Formula/opencascade.rb

```diff
@@ -0,0 +1,45 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+  head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "freeimage"
+  depends_on "tbb"
```

Comment:
non-build deps need to be alphabetized amongst themselves

Reply:
Will fix.  Seems like another candidate for `brew audit --strict` ;)

Reply:
not really because there are cases where that's not actually possible

Reply:
too bad because I'm sure you are getting tired of telling *newbs* to alphabetize their depends ;)

Reply:
I find they usually learn the alphabet relatively quickly.